src/applications/personalization/search-representative/config/chapters/search/searchRepresentativeReviewWidget.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+const SearchRepresentativeReviewWidget = props => {
+  const capitalizeFirstLetter = string => {
+    return string.charAt(0).toUpperCase() + string.slice(1);
+  };
+
+  return (
+    <>
+      {Object.keys(props?.formData?.representativeData).map((item, index) => {
+        return (
+          <>
```

Comment:
Also here you might want to use:
```jsx
<React.fragment key={index}>
...
</React.fragment>
```
because it thinks the children don't have 'key' props if you do it on the div